handlers/guest_links_test.go

```diff
@@ -346,3 +345,193 @@ func TestDeleteInvalidGuestLink(t *testing.T) {
 		t.Fatalf("DELETE returned wrong status code: got %v want %v", status, http.StatusBadRequest)
 	}
 }
+
+func TestGuestLinkToggleInValidRequest(t *testing.T) {
+	type testCase struct {
+		description      string
+		operationUrls    []string
+		guestLinkInStore picoshare.GuestLink
+		expected         picoshare.GuestLink
+	}
+
+	for _, tt := range []testCase{
+		{
+			description: "guest_link operation: disable",
```

Comment:
Can we make the descriptions be what we are saying the endpoint does. For example, for this one, I'd say, "disabling active guest link succeeds"